deb/common/control

```diff
@@ -64,3 +65,20 @@ Description: Docker CLI: the open-source application container engine
  language, framework or packaging system. That makes them great building blocks
  for deploying and scaling web apps, databases, and backend services without
  depending on a particular stack or provider.
+
+Package: docker-ce-rootless-extras
+Architecture: linux-any
+Depends: docker-ce, ${shlibs:Depends}
+Conflicts: rootlesskit
+Replaces: rootlesskit
+Breaks: rootlesskit
```

Comment:
Is `rootlesskit` in any of the distro packages? Should we name our binary different? (`docker-rootlesskit`)?

Reply:
Debian 11 has: https://packages.debian.org/bullseye/rootlesskit

I don't think we should add prefix. The `docker-` prefix in `docker-containerd` and `docker-runc` had gone away.